source/develop/release-management/features/hot-unplug-memory.html.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Hot Unplug Memory
+category: feature
+authors: mzamazal
+wiki_category: Feature
+wiki_title: Features/Hot Unplug Memory
+wiki_revision_count: 1
+wiki_last_updated: 2017-02-14
+---
+
+# Hot unplug memory
+
+## Summary
+
+This feature allows removing memory from a VM while the VM is running.
+
+## Owner
+
+*   Name: Milan Zamazal
+*   Email: mzamazal@redhat.com
+*   BZ: https://bugzilla.redhat.com/1228543
+
+## Description
+
+Until now, it was possible to add memory only to a running VM.  In order to
+remove memory from a VM, the VM had to be shut down.
+
+It is now possible to hot unplug memory from a running VM.  However there are
+some limitations:
+
+- You can't remove arbitrary amount of memory.  Only previously hot plugged
+  memory devices can be removed.
+
+- The guest OS must support memory hot unplug.  Most contemporary Linux systems
+  should support it.
+
+- The previously hot plugged memory must be movable.
```

Comment:
How about adding: I.e. All blocks of the hot plugged memory device must have been onlined as `online_movable`.?

This would a bit explain what a movable memory is as well as that memory devices consists of blocks.